### ISSUE TITLE
DragOverlay - Cannot read property 'then' of undefined

### ISSUE THREAD

**Issue by oliverfindl** [clauderic/dnd-kit#105]
Hello,

I was trying to fiddle with this package and followed instructions for [Sortable example](https://docs.dndkit.com/presets/sortable). At [final step](https://docs.dndkit.com/presets/sortable#drag-overlay), after adding `DragOverlay` it crashes.

![image](https://user-images.githubusercontent.com/33978096/109389903-d4b3fa80-790e-11eb-8a7d-37cd0a6fc08a.png)

After removing `DragOverlay` it works again. 

Couldn't find this bug in issues, so I'm reporting it.

Thanks.

**Comment by ranbena** [comment]
@oliverfindl a few questions:
1. Do the [Storybook Sortable Examples](https://5fc05e08a4a65d0021ae0bf2-zditmhymjz.chromatic.com/?path=/story/presets-sortable-vertical--basic-setup) generate the same error for you?
  If not, the reason for the error might be in the way you implemented it. Perhaps share a reproducible code example.

2. What browser version are you testing with? Some do not support [Animation.finished](https://developer.mozilla.org/en-US/docs/Web/API/Animation/finished) which dnd-kit relies on.

**Comment by oliverfindl** [comment]
Hello,

Storybook example is also broken for me.

I'm using opera 68.0.3618.173 (latest version in repo of my OS, but not latest version of Opera) that should be equivalent to Chrome 81.0.4044.138. Just tested in newest Chrome and Firefox, and its working there. According [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Animation/finished#browser_compatibility) and [CIU](https://caniuse.com/mdn-api_animation_finished), `Animation.finished` is not even supported in Chrome (although its working for me), so guess, its not ready for production use (yet).

Anyway, I found out [this polyfill](https://github.com/web-animations/web-animations-js) which solved issue for me.

Closing this issue.

Thanks.
